apps/test/unit/templates/rubrics/RubricFloatingActionButtonTest.jsx

```diff
@@ -134,68 +134,71 @@ describe('RubricFloatingActionButton - React Testing Library', () => {
       );
       const image = screen.getByRole('img', {name: 'AI bot'});
       fireEvent.load(image);
-      const fab = screen.getByRole('button', {name: 'AI bot'});
-      expect(fab.classList.contains('unittest-fab-pulse')).to.be.false;
-      sendEventSpy.restore();
+      const fab = screen.getByRole('button', {
+        name: i18n.openOrCloseTeachingAssistant(),
+      });
+      expect(fab.classList.contains('unittest-fab-pulse')).toBe(false);
     });
   });
-});
 
-describe('RubricFloatingActionButton - Enzyme', () => {
   it('begins closed when student level info is null', () => {
-    const wrapper = shallow(<RubricFloatingActionButton {...defaultProps} />);
-    expect(wrapper.find('Connect(RubricContainer)').length).to.equal(1);
-    expect(wrapper.find('Connect(RubricContainer)').props().open).to.equal(
-      false
+    render(
+      <Provider store={store}>
+        <RubricFloatingActionButton {...defaultProps} />
+      </Provider>
     );
-  });
-
-  it('begins open when student level info is provided', () => {
```

Comment:
this test name is a lie - the test was only passing because we weren't cleaning session storage between test cases.